deployments/helm-chart/templates/controller-leader-election-configmap.yaml

```diff
@@ -5,3 +5,7 @@ metadata:
   namespace: {{ .Release.Namespace }}
   labels:
     {{- include "nginx-ingress.labels" . | nindent 4 }}
+{{- if .Values.controller.reportIngressStatus.annotations }}
```

Comment:
Hi @RubyLangdon 

Just double checking that the spinnaker annotations are needed to be configured for the leader election configmap. This config map is used internally by the Ingress Controller for leader election to work. The IC never changes its `data` field, it only sets the annotation `control-plane.alpha.kubernetes.io/leader:`

Reply:
In some situations I see spinnaker append increasing version numbers to configmaps when they haven't changed since the last deployment. I'm not sure why but I'd prefer to be able to stop versioning to be safe. Also, even if that doesn't happen the configmaps always get `v000` appended to them the first time they are deployed, which we don't necessarily want.